### ISSUE TITLE
Running four queries on each submit in editor

### ISSUE THREAD

**Issue by joente** [thingsdb/ThingsGUI#27]
Each submit in the editor creates four queries while just one query is expected.

For example, a simple query like `"Hello ThingsDB";` create these three additional queries:

```
QUERY: thing(.id());
QUERY: procedures_info()
QUERY: types_info()
```

**Comment by joente** [comment]
Instead of running the queries on each "submit", is it possible to get "refresh" icons on the **procedures** and **types** list?

**Comment by AnjaBruls** [comment]
I removed all additional queries. Procedures and types are refreshed using the refresh button or on opening the panels. This fix will be released in v0.1.3.

**Comment by AnjaBruls** [comment]
Implemented in release v0.1.3
